configs/webpack-config-compass/src/index.ts

```diff
@@ -111,6 +111,14 @@ const sharedResolveOptions = (
       // Some lg test helpers that are getting bundled due to re-exporting from
       // the actual component packages, never needed in the webpack bundles
       '@lg-tools/test-harnesses': false,
+
+      // The react/jsx-runtime can fail the import when imported from an esm
+      // module because React 17 doesn't have ./jsx-runtime specified in the
+      // package.json `exports` field. This issue will go away if we can update
+      // to React 18, but for the time being we will just resolve react to
+      // actual files ourselves to work around that
+      'react/jsx-runtime': require.resolve('react/jsx-runtime'),
+      react: require.resolve('react'),
```

Comment:
Probably should add this to ignore in depcheckrc file, we do this already for another package that we just need to resolve

Reply:
with support for cjs, i ended up removing this.